x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -367,6 +349,28 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
         signal = { name: null };
       }
 
+      const configPatternList = coreStart.uiSettings.get(DEFAULT_INDEX_KEY);
```

Comment:
I thought that the creation of the security solution data view will be created here if does not exist. I thought that might simplify a lot of code.

Reply:
thats whats happening on line 357, it all happens servers side